internal/docs/markdown.go

```diff
@@ -112,6 +113,7 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 	if err := printOptions(w, cmd); err != nil {
 		return err
 	}
+	fmt.Fprint(w, "{% endraw %}\n")
```

Comment:
Seems like it would be a bit simpler if we just move this end tag down to line 129 and remove the `raw` tags from the example section.

Reply:
Ah I missed that, ignore my comment then.